docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,99 @@
+            - create a list of non-matching capacity validation points: `non_matching_capacity_validation_points = []`
+            - create a list of missing capacity validation points: `missing_capacity_validation_points = []`
+            - loop through the expected_chwt_temps: `for cwt in expected_cwt_temps:`
+                - loop through each expected_ecwt_temps: `for ect in expected_ecwt_temps:`
+                    - look for the capacity validation point in capacity_validation_pts_dict: `if capacity_validation_pts_dict[[cwt, ect]]:`
+                        - calculate the expected capacity by multiplying the result of the formula by the rated_capacity: `expected_capacity = (cap_f_t_coefficients[0] + cap_f_t_coefficients[1] * cwt + cap_f_t_coefficients[2] * cwt^2 + cap_f_t_coefficients[3] * ect + cap_f_t_coefficients[4] * ect^2 + cap_f_t_coefficients[5] * cwt * ect) * rated_capacity`
+                        - get the given capacity: `given_capacity = capacity_validation_pts_dict[[cwt, ect]]`
+                        - add the given capacity to the given_capacities dictionary: `given_capacities[[cwt,ect]] = given_capacity`
+                        - Compare the expected capacity to the given capacity.  This should not be an exact match, but with a margin of error (see notes at the bottom for suggested margins of errors).  We don't need to do anything if the capacities match, but if they don't match we need to add the conditions to the non_matching_capacity_validation_points list: `if expected_capacity != given_capacity: non_matching_capacity_validation_points.append({"CHWT": cwt, "ECWT": ect})`
+                    - otherwise this value doesn't exist, append these conditions to the missing_capacity_validation_points list: `else: missing_capacity_validation_points.append({"CHWT": cwt, "ECWT": ect})`
+
+            - the following lines or logic do the power validation check:
+            - create a list of non-matching power validation points: `non_matching_power_validation_points = []`
+            - create a list of missing power validation points: `missing_power_validation_points = []`
+            - loop through the expected_cwt_temps: `for cwt in expected_cwt_temps:`
+                - loop through each expected_ecwt_temps: `for ect in expected_ecwt_temps:`
+                    - look for the power validation points in power_validation_pts_dict: `if power_validation_pts_dict[[cwt, ect]]:`
+                        - we are expecting to see multiple validation points aligning with the expected_validation_plr.  Create a list of part load ratios that are given.  Later we'll compare this with the expected list to make sure that all points are given: `given_plrs = []`
+                        - look at each power validation point in the list: `for power_validation_point_result in power_validation_pts_dict[[cwt, ect]]:`
+                            - get the load: `load = power_validation_point.load`
```

Comment:
I may just be tired but I am not following how this load is getting retrieved from this. I don't think you are looping through power_validation_point objects at this point.

Reply:
load is a property of [ChillerPowerValidationPoint](https://github.com/open229/ruleset-model-description-schema/blob/develop/docs229/ASHRAE229.schema.md#chillerpowervalidationpoint)

Reply:
Okay but I do not see how you are getting to this with the current logic. How are you able to reference the power_validation_point, you are not looping through them anywehre and there can be multiple associated with the chiller.  power_validation_point is an object so I am not following how you are able to reference it here.

Reply:
you're right - I made changes also on line 50, so that power_validation_pts_dict[dict_key] references power_validation_points, not a list of power_validation_point results.  Then we can get both the result and load on lines 75 and 76